shub/login.py

```diff
@@ -1,21 +1,56 @@
-import os, re, click
-from shub.utils import get_key_netrc, NETRC_FILE
+import re
+import click
+import os
+from shub import auth
+from scrapy.utils.conf import get_config
```

Comment:
I'm unsure about this now... Scrapy brings a lot of dependencies with it, but it's weird to require all that just for a few functionality.

@pablohoffman @josericardo would you be okay with duplicating the functions from scrapy.utils.conf here?